ros__parameters:
    Kp_steering : 0.8 
    Ki_steering : 0.0 
    Kd_steering : 0.0 
    zero_throttle  : 0 
    max_throttle : 0.4
    min_throttle : 0.3
    error_threshold : 0.16 
    max_right_steering : 0.75
    max_left_steering : -0.75